libr/bin/bin.c

```diff
@@ -1401,7 +1401,7 @@ R_API RList *r_bin_get_mem(RBin *bin) {
 
 R_API int r_bin_is_big_endian(RBin *bin) {
 	RBinObject *o = r_bin_cur_object (bin);
-	return o? o->info->big_endian: false;
+	return o? o->info->big_endian: -1;
```

Comment:
is_big_endian denotes boolean. arent we defaulting to little endian? we can just return `false` here